setup.py

```diff
@@ -1,16 +1,17 @@
 from setuptools import find_packages, setup
 
 setup(
-    name='mozilla-taar',
-    use_scm_version=True,
+    name='mozilla-taar2',
+    use_scm_version=False,
+    version='0.0.18',
     setup_requires=['setuptools_scm', 'pytest-runner'],
     tests_require=['pytest'],
     include_package_data = True,
     packages=find_packages(exclude=['tests', 'tests/*']),
     description='Telemetry-Aware Addon Recommender',
-    author='Mozilla Foundation',
-    author_email='fx-data-dev@mozilla.org',
-    url='https://github.com/mdoglio/taar',
+    author='Victor Ng',
```

Comment:
I'm not entirely sure why this should be *Victor Ng* rather than *Mozilla Foundation* :) The repo below points to a `mozilla/taar`.

Reply:
I changed it mostly so that we have an actual contact person who can answer questions.  The license owner ends up being the Mozilla Corporation for any of our work.